docs/content/manual/manual.yml

```diff
@@ -1148,15 +1148,15 @@ sections:
           (except it does not output the empty list, representing .
           itself).
 
-          `paths(f)` outputs the paths to any values for which `f` is true.
-          That is, `paths(numbers)` outputs the paths to all numeric
+          `paths(f)` outputs the paths to any values for which `f` is `true`.
+          That is, `paths(type == "number")` outputs the paths to all numeric
```

Comment:
This got me thinking that i several times wanted to have `is_number` etc functions, for fq i have these https://github.com/wader/fq/blob/d15a41f9430f869dc6d11ac360f56c29e3a1b45e/pkg/interp/internal.jq#L147C1-L153 because i kept needing it

Reply:
`def paths(node_filter): . as $dot|paths|select(. as $p|$dot|getpath($p)|node_filter);`

`f` doesn't have to output `true` -- it just has to not be `empty` or `null`.

Reply:
I mean, maybe that's a bug...

But separately from that, jq considers any non-null, non-empty value to be "trueish".

Reply:
and non-false. :^)

Reply:
Also empty is not really a falsish value, it really stops execution for that expression. e.g. `if empty then "yes" else "no" end` outputs nothing, not `"no"`. Anyway, this is all intended in my opinion. `false` and `null` are the only false values, and `empty` works as it should.

Reply:
Do want to start using wording like "... for which `f` is considered true" or "... for which `f` is trueish" and have some separate section about jq and truthiness? "truthiness" i currently mentioned in the docs for if-then-else

Reply:
As a user manual, it is enough to mention that the filter works well with boolean filter arguments. No need to mention all its behaviors.

Reply:
Yes, speaking of "empty values" is not exactly accurate.  It's a bit of a colloquialism in jq speak.

Reply:
@wader I'm not sure that it's a good idea to document all the places where trueish values work, yeah.

Reply:
@nicowilliams  true :)

Reply:
ish